docs/content/preview/explore/multi-region-deployments/asynchronous-replication-ysql.md

```diff
@@ -34,119 +34,126 @@ type: docs
+   | JDBC                : jdbc:postgresql://127.0.0.1:5433/yugabyte?user=yugabyte&password=yugabyte  |
+   | YSQL                : bin/ysqlsh   -U yugabyte -d yugabyte                                       |
+   | YCQL                : bin/ycqlsh   -u cassandra                                                  |
+   | Data Dir            : /Users/myuser/yugabyte-2.7.2.0/datacenter-east/data                        |
+   | Log Dir             : /Users/myuser/yugabyte-2.7.2.0/datacenter-east/logs                        |
+   | Universe UUID       : 4fb04760-4b6d-46a7-83cf-a89b2a056579                                       |
+   +--------------------------------------------------------------------------------------------------+
+   🚀 yugabyted started successfully! To load a sample dataset, try 'yugabyted demo'.
+   🎉 Join us on Slack at https://www.yugabyte.com/slack
+   👕 Claim your free t-shirt at https://www.yugabyte.com/community-rewards/
+   ```
+   
+1. Create and start your second local cluster that simulates "Data Center = West" by running the following `yugabyted start` command from your YugabyteDB home directory:
+
+     ```sh
+   ./bin/yugabyted start --base_dir=datacenter-west --listen=127.0.0.2
+   ```
+   
+    The preceding command starts a one-node cluster using IP address of `127.0.0.2` and creates `datacenter-west` as the base directory. Expect to see an output similar to the following:
+
+   
+
+     ```output
+   Starting yugabyted...
+   ✅ System checks
+   
+   +--------------------------------------------------------------------------------------------------+
+   |                                            yugabyted                                             |
+   +--------------------------------------------------------------------------------------------------+
+   | Status              : Running. Leader Master is present                                          |
+   | Web console         : http://127.0.0.2:7000                                                      |
+   | JDBC                : jdbc:postgresql://127.0.0.2:5433/yugabyte?user=yugabyte&password=yugabyte  |
+   | YSQL                : bin/ysqlsh -h 127.0.0.2  -U yugabyte -d yugabyte                           |
+   | YCQL                : bin/ycqlsh 127.0.0.2 9042 -u cassandra                                     |
+   | Data Dir            : /Users/myuser/yugabyte-2.7.2.0/datacenter-west/data                        |
+   | Log Dir             : /Users/myuser/yugabyte-2.7.2.0/datacenter-west/logs                        |
+   | Universe UUID       : ad78f70c-0741-4c7e-b610-315d55d7f248                                       |
+   +--------------------------------------------------------------------------------------------------+
+   🚀 yugabyted started successfully! To load a sample dataset, try 'yugabyted demo'.
+   🎉 Join us on Slack at https://www.yugabyte.com/slack
+   👕 Claim your free t-shirt at https://www.yugabyte.com/community-rewards/
+     ```
+
```

Comment:
Yeah, most of the file is the same. It's too bad that our docs are organized this way. The _ycql file wasn't the focus of this ticket, and I prettied up a little bit, but can't allocate any more time because I need to switch to the platform asap. I'll ask Marco if the Add table section should also be in _ycql and if so, will create a new ticket.